### ISSUE TITLE
python faker library integration

### ISSUE THREAD

**Issue by eracle** [aleju/face-generator#3]
Hi there!
Is in the roadmap any integration with the python faker library? :-D

**Comment by aleju** [comment]
Not really. I don't know much about the faker library and the face-generator project is now quite old, so I prefer to work on the newer projects.
